### ISSUE TITLE
[PHP] New question with answer

### ISSUE THREAD

**Issue by VIDAL-Antoine** [Ebazhanov/linkedin-skill-assessments-quizzes#6947]
- [x] New question 

#### Question. What is the output of this script?

```php
$believable = 'false';
$myth = 'The moon is made of green cheese';
$calc = 10**3+1;
if ($believable) {
    echo $myth;
}
else {
    echo $calc;
}
```

- [ ] 10000
- [ ] 31
- [ ] 1001
- [x] The moon is made of green cheese

Explanation : 'false' evaluates to true since it is a string so the if condition is met.

![php-quiz](https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/assets/71827734/96d3b16c-23e2-47cc-9668-b6be925a2df1)

![php-quiz-proof](https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/assets/71827734/c81fab80-8880-418b-bfd4-40716b6faaa3)

**Comment by haxkd** [comment]
solved #6948

**Comment by aaron-muti-420** [comment]
Is this issue still open?
